.gitignore

```diff
@@ -21,3 +21,5 @@ pnpm-debug.log*
 *.njsproj
 *.sln
 *.sw?
+
+.env
```

Comment:
I would move that with the other `.env` files. 

And I think `/dist` will still be nested in `ui`? Not really sure how it works with yarn workspaces